ydb/__init__.py

```diff
@@ -18,6 +18,7 @@
 from .import_client import *  # noqa
 from .tracing import *  # noqa
 from .topic import *  # noqa
+import ydb.draft  # noqa
```

Comment:
```suggestion
import .draft  # noqa
```

Reply:
fixed to `from .draft impot *` (your suggestion doesn't work)